### ISSUE TITLE
Why do open web pages display 'Not Found'?

### ISSUE THREAD

**Issue by Bennie61** [voxel51/fiftyone#3535]
```
# my code
import fiftyone as fo
import fiftyone.zoo as foz

if __name__ == '__main__':
    dataset = foz.load_zoo_dataset("quickstart")
    session = fo.launch_app(dataset, port=8103)
    session.wait()

```
fiftyone version: 0.21.6

### Describe the problem
I follow and run the quickStart code，but the open page display 'Not Found'. I have changed the port, but didn't work.
![image](https://github.com/voxel51/fiftyone/assets/76785077/5a3a4d3f-d9b0-4847-bcde-d263978b8fb9)

**Comment by benjaminpkane** [comment]
Hi @Bennie61. Downgrading to `0.21.4` until `0.22.0` is released may solve your issue. https://github.com/voxel51/fiftyone/pull/3439 has the details

**Comment by benjaminpkane** [comment]
Resolved with `0.22.0`!
